### ISSUE TITLE
Derivation for Forward pass and back pass of Cross Entropy loss function

### ISSUE THREAD

**Issue by azaher1215** [COD1995/ml-meta#2]

**Comment by COD1995** [comment]
Sounds good, I can't wait to see the progress.
